pr_watch/migrations/0008_update_organization.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2018-08-24 20:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pr_watch', '0007_watchedfork_openedx_release'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='watchedfork',
+            name='organization'
+        ),
+        migrations.AddField(
+            model_name='watchedfork',
+            name='organization',
+            field=models.ForeignKey(null=True, help_text='Organization to watch. PRs against the watched fork made by members of this organization will trigger a sandbox build.', on_delete=django.db.models.deletion.CASCADE, to='userprofile.Organization'),
```

Comment:
This was failing on stage, because there was data for the index. So I had to alter the migrations, so that it first removes the organization field completely, and then adds it again.

Reply:
@rocioar Ah, OK -- so this migration (`pr_watch/migrations/0008_update_organization.py`) now takes care of removing the `organization` field from the `WatchedFork` model and adding it back`. What's the purpose of `pr_watch/migrations/0009_organization_db_index.py`, do we still need it?